spirv_glsl.cpp

```diff
@@ -5016,6 +5039,22 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 			require_extension("GL_OES_standard_derivatives");
 		break;
 
+	case OpDPdxFine:
+		UFOP(dFdxFine);
```

Comment:
There's an extension for 4.0 (https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_derivative_control.txt) and it's core for 4.5. But nothing for ES I think.